src/routes/dev/data-graphic/components.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+  import { Axis } from "$lib/components/data-graphic/guides";
+  import { SimpleDataGraphic } from "$lib/components/data-graphic/elements";
+  import type { AxisSide } from "$lib/components/data-graphic/guides/types";
+  const width = 132;
+  const height = 132;
+  const leftRight = 36;
+  const topBottom = 24;
+  const sides: AxisSide[] = ["left", "right", "top", "bottom"];
+</script>
+
+<h1 class="text-xl">Graphic Components</h1>
+
+<h2 class="text-lg">Axis</h2>
+<section>
+  {#each sides as axis}
+    <div>
+      side=<i>"{axis}"</i>
+      <div class="border border-gray-200 rounded">
+        <SimpleDataGraphic
+          xType="number"
+          yType="number"
+          {width}
+          {height}
+          left={leftRight}
+          right={leftRight}
+          top={topBottom}
+          bottom={topBottom}
+          xMin={0}
+          xMax={100}
+          yMin={0}
+          yMax={100}
```

Comment:
Circling back to one of the previous comment made on `Line.svelte`, I think these values should be automatically set by the data itself without the need for marks

Reply:
agreed, though it's pretty common to need to set the min and max values manually if the plot bounds are not actually conveying what's needed.